src/main/java/com/Futurecollars/Lekcja7/Zadanie7/Training.java

```diff
@@ -0,0 +1,96 @@
+package com.Futurecollars.Lekcja7.Zadanie7;
+
+public class Training {
+    int duration;
+    int burnedCalories;
+    int averageHeartRate;
+
+    public Training(int duration, int burnedCalories, int averageHeartRate) {
+        this.duration = duration;
+        this.burnedCalories = burnedCalories;
+        this.averageHeartRate = averageHeartRate;
+    }
+
+    public int getDuration() {
+        return duration;
+    }
+
+    public int getBurnedCalories() {
+        return burnedCalories;
+    }
+
+    public int getAverageHeartRate() {
+        return averageHeartRate;
+    }
+
+    public int gradeDuration(int duration)
+    {
+        if (duration < 30) {
+            return 1;
+        } else if (duration <= 60) {
+            return 2;
+        } else {
+            return 3;
+        }
+    }
+    public int gradeBurnedCalories(int burnedCalories)
+    {
+        if (burnedCalories < 300) {
+            return 1;
+        } else if (burnedCalories <= 400) {
+            return 2;
+        } else {
+            return 3;
+        }
+    }
+    public int gradeAverageHeartRate(int averageHeartRate) {
+        if (averageHeartRate < 160) {
+            return 3;
+        } else if (averageHeartRate <= 175) {
+            return 2;
+        } else {
+            return 1;
+        }
+    }
+    public double calculateScaleAverage(int gradeDuration, int gradeBurnedCalories, int gradeAverageHeartRate) {
```

Comment:
dlaczego ta metoda ma parametry wchodzące skoro w konstruktorze przekazujesz te parametry inicjalizując cały obiekt. Masz do nich dostęp poprzez this.duration itp